extending-valtimo/extending-valtimo.md

```diff
@@ -1,5 +1,9 @@
 # Extending Valtimo
 
+This section contains information on how to use Valtimo features for which additional code is required. These are 
+usually features that have been set up in a way so that the default functionality can be easily extended. This section 
+is intended for people who want to get more out of Valitmo and are able to write Java or Kotlin code.
```

Comment:
Valitmo -> Valtimo.

Maybe mention here all examples will be given in Java?

Reply:
And maybe give an example on what kind of stuff to expect in this section (e.g. creating custom form field data resolvers).

Reply:
> Maybe mention here all examples will be given in Java?

Examples currently are in java or kotlin. These languages are already mentioned. How would you want to do this?